Main() {
    try {
        throw 1;
    } catch(...) {
        return true;
    }
}